packages/react-scripts/package.json

```diff
@@ -48,13 +48,13 @@
     "postcss-flexbugs-fixes": "3.0.0",
     "postcss-loader": "2.0.5",
     "promise": "7.1.1",
-    "react-dev-utils": "^3.0.0",
-    "react-error-overlay": "^1.0.7",
-    "style-loader": "0.17.0",
-    "sw-precache-webpack-plugin": "0.9.1",
-    "url-loader": "0.5.8",
-    "webpack": "2.6.1",
-    "webpack-dev-server": "2.4.5",
+    "react-dev-utils": "^3.0.2",
```

Comment:
This is our own module. Please don't bump it now. It is bumped during the release.